mcux/drivers/imx/CMakeLists.txt

```diff
@@ -25,3 +25,7 @@ zephyr_sources_ifdef(CONFIG_ETH_MCUX          fsl_enet.c)
 zephyr_sources_ifdef(CONFIG_ENTROPY_MCUX_TRNG fsl_trng.c)
 zephyr_sources_ifdef(CONFIG_PWM_MCUX fsl_pwm.c)
 zephyr_sources_ifdef(CONFIG_VIDEO_MCUX_CSI    fsl_csi.c)
+zephyr_sources_ifdef(CONFIG_DMA_MCUX_EDMA  fsl_edma.c)
+zephyr_sources_ifdef(CONFIG_DMA_MCUX_EDMA  fsl_dmamux.c)
+zephyr_sources_ifdef(CONFIG_IMX_SAI        fsl_sai.c)
+zephyr_sources_ifdef(CONFIG_IMX_SAI_DMA    fsl_sai_edma.c)
```

Comment:
I don't remember seeing an SAI driver in your zephyr PR. Do you still need to build these files?

Reply:
my next plan is to enable SAI. @MaureenHelm.

Reply:
The driver should probably be called `CONFIG_I2S_MCUX_SAI`

Reply:
ok